.drone.yml

```
@@ -176,16 +176,13 @@ steps:
         - tag
 
 ---
-name: e2e-kubernetes-1.26
+name: e2e-kubernetes-1.27
 kind: pipeline
 type: docker
 
```

You add a directive to clone the last commit instead of the full repo history:
```suggestion
clone:
  depth: 1
```